.github/workflows/lint.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           MARKDOWN_CONFIG_FILE: .markdownlint.json
           # Always check all the content
-          VALIDATE_ALL_CODEBASE: true
+          VALIDATE_ALL_CODEBASE: false
```

Comment:
Why are you changing the configuration of the linter in this PR?

Reply:
I have not changed this file or added it to the branch, I assumed this was a change made on the PR creation to save run time on the tests.
But as I didn't make this change happy for it to change back

Reply:
This change is clearly in one of the commits you are pushing with the PR :)
But maybe it's a left over due to the fact that the source branch you used wasn't maybe properly synchronised with the main branch?

Reply:
Odd, I have reverted this now